lib/avalanche/steps/get_partitions.ex

```diff
@@ -75,10 +75,18 @@ defmodule Avalanche.Steps.GetPartitions do
 
   # reuse current request and turn it into a `StatusRequest`
   defp build_status_request(%Req.Request{} = request, path, partition, row_types) do
+    %{path: path} = URI.parse(path)
+
+    url =
+      request.url
+      |> URI.parse()
+      |> Map.put(:path, path)
+      |> Map.put(:query, "async=false&partition=#{partition}")
```

Comment:
I think you want to remove this. `partition` is added below and `async=false` is the default.

Also, I think `async` can be passed by the user as an option so we don't want to force it.

Reply:
Oops, good catch 👍🏼 thanks!